src/fsharp/xlf/FSComp.txt.de.xlf

```diff
@@ -5438,7 +5438,7 @@
         <note />
       </trans-unit>
       <trans-unit id="fscBadAssemblyVersion">
-        <source>An {0} specified version '{1}', but this value is invalid and has been ignored</source>
+        <source>The attribute {0} specified version '{1}', but this value is invalid and has been ignored</source>
         <target state="translated">Ein {0} gab Version '{1}' an, dieser Wert ist jedoch ungültig und wurde ignoriert.</target>
```

Comment:
Honestly this sounds just as bad as the previous English version. Should I change that as well or is somebody adapting all translations?

/cc @forki because I think he has fought for a better translation before.